src/main.c

```diff
@@ -496,12 +624,12 @@ char aes_encrypt(unsigned char *input,
     else
     {
         /* Set the block values, for the block:
-         * a0,0 a0,1 a0,2 a0,3
-         * a1,0 a1,1 a1,2 a1,3
-         * a2,0 a2,1 a2,2 a2,3
-         * a3,0 a3,1 a3,2 a3,3
-         * the mapping order is a0,0 a1,0 a2,0 a3,0 a0,1 a1,1 ... a2,3 a3,3
-         */
+        * a0,0 a0,1 a0,2 a0,3
+        * a1,0 a1,1 a1,2 a1,3
+        * a2,0 a2,1 a2,2 a2,3
+        * a3,0 a3,1 a3,2 a3,3
+        * the mapping order is a0,0 a1,0 a2,0 a3,0 a0,1 a1,1 ... a2,3 a3,3
+        */

```

Comment:
Avoid changing the indentation in code that hasn't been modified, as it makes it harder to track the main changes in the review.